scipy/stats/_stats.pyx

```diff
@@ -753,3 +753,90 @@ def gaussian_kernel_estimate(points, values, xi, precision, dtype, real _=0):
                 estimate[j, k] += values_[i, k] * arg
 
     return np.asarray(estimate)
+
+
+@cython.wraparound(False)
+@cython.boundscheck(False)
+def gaussian_kernel_estimate_log(points, values, xi, precision, dtype, real _=0):
+    """
+    def gaussian_kernel_estimate_log(points, real[:, :] values, xi, precision)
+
+    Evaluate the log of the estimated pdf on a provided set of points.
+
+    Parameters
+    ----------
+    points : array_like with shape (n, d)
+        Data points to estimate from in ``d`` dimensions.
+    values : real[:, :] with shape (n, p)
+        Multivariate values associated with the data points.
+    xi : array_like with shape (m, d)
+        Coordinates to evaluate the estimate at in ``d`` dimensions.
+    precision : array_like with shape (d, d)
+        Precision matrix for the Gaussian kernel.
```

Comment:
@europeanplaice if I'm reading this correctly, `precision` is the `inv_cov` matrix passed in during the call to `logpdf`. gh-5047 was attempting to avoid the use of `inv_cov`. Do you have a moment to take a look at what that PR was trying to do and comment on whether that is compatible with what you're doing here? I was going to take over and complete that PR, but I didn't want to cause merge conflicts here.

Reply:
Hi, @mdhaber. Do you mean not https://github.com/scipy/scipy/issues/5047 but https://github.com/scipy/scipy/pull/5087?

Reply:
Yes, thanks! Sorry about that.

Reply:
@europeanplaice I'd suggest finishing this PR without worrying about this extra enhancement, then we can do it as a follow-up. 
Is there anything you need from us to finish this, btw?

Reply:
@mdhaber 
For me it's good to finish this PR at this time. I don't have anything to add to it. Thanks for your suggestion.

Reply:
Ok, but it looks like there are some comments above (https://github.com/scipy/scipy/pull/15493#pullrequestreview-988546456) that don't have responses. Could you reply to those so we can ask Ralf to take a final look?